tests/tpch/generate_plot.py

```diff
@@ -1,10 +1,18 @@
+import warnings
+
 import altair as alt
 import click
 import pandas as pd
 
 
 def generate(outfile="chart.json", name=None, scale=None):
-    df = pd.read_sql_table(table_name="test_run", con="sqlite:///benchmark.db")
+    try:
+        df = pd.read_sql_table(table_name="test_run", con="sqlite:///benchmark.db")
+    except ValueError:
+        warnings.warn(
+            "test_run table not found, run with --benchmark to generate charts"
+        )
+        return
```

Comment:
Probably the `make_chart` fixture should check if the `--benchmark` keyword was engaged, and if not, return quickly.

Reply:
Done